test/spec/modules/userId_spec.js

```diff
@@ -434,14 +436,17 @@ describe('User ID', function() {
           }, {
             name: 'intentIqId',
             storage: { name: 'intentIqId', type: 'cookie' }
+          }, {
+            name: 'zeotapIdPlus',
+            storage: { name: 'IDP', type: 'cookie' }
```

Comment:
per above, this can probably be dropped
```suggestion
            name: 'zeotapIdPlus'
```

Reply:
dropped.